src/web-ui/src/public/Welcome.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <Layout :showNav="false" :showFooter="false" :backgroundColor="'var(--aws-squid-ink)'">
+    <div class="container mb-2 text-left">
+      <header class="px-3 px-md-5">
+        <h1 class="heading my-5 text-center">Welcome to the Retail Demo Store!</h1>
+
+        <p>
+          The Retail Demo Store is a demo platform to demonstrate how AWS infrastructure and services can be used to
+          build personalized customer experiences across multiple channels of engagement.
+        </p>
+
+        <p>
+          The Retail Demo Store mimics an online store with products, users and user-interactions to showcase the
+          personalized product recommendations a shopper would receive based on historical and real time click-through
+          behavior.
+        </p>
+
+        <div class="mt-2 d-flex flex-column flex-lg-row align-items-lg-center">
+          <div class="mb-2 mb-lg-0">AWS Services Included:</div>
+
+          <ul
+            class="service-list mb-0 pl-0 flex-grow-1 d-flex flex-column flex-lg-row justify-content-lg-around font-weight-bold"
+          >
+            <li v-for="service in services" :key="service.text" class="mb-2 mb-lg-0">
+              <ServiceBadge :text="service.text" :src="service.src"></ServiceBadge>
+            </li>
+          </ul>
+        </div>
+
+        <!-- make both links to go /auth until we have dedicated sign in and create account pages -->
```

Comment:
The Retail Demo Store has certain functionality that does not need sign in such viewing products/similar items , checkiing out as guest or accessing the demo guide. Do we need to add a "Skip this step" button?

Reply:
Definitely makes sense to have that. We've opted to add a "skip this step" button to the roadmap and will make sure to add it before merging into the main branch.